### PR TITLE
minor: prevent find running for quarterly cleanup if it doesn't exist

### PR DESCRIPTION
**Pull request by rnveach**
Prevent build failures on other PRs.

### CODE REVIEW

File: .ci/travis.sh [hunk 1]
```diff
@@ -139,7 +142,13 @@ jacoco)
   ;;
 
 quarterly-cache-cleanup)
-  find ~/.m2 -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
+  MVN_REPO=$(mvn -e --no-transfer-progress help:evaluate -Dexpression=settings.localRepository \
+    -q -DforceStdout);
+  if [[ -d ${MVN_REPO} ]]; then
+    find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
```
Comment (nrmancuso) [inline .ci/travis.sh:148]: I would prefer a simply appending `|| true`, if cache is where we expect, OK. If not, download dependencies and run maven. Adding another maven command just adds fragility and makes for a new point of failure.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: .ci/travis.sh (hunk 1 above)
Comment (rnveach) [reply .ci/travis.sh:148]: if `find` errors out, will it display that with `|| true`? Also do you want me to remove the display from the init too?

Comment (nrmancuso) [reply .ci/travis.sh:148]: >if find errors out, will it display that with || true?

yes:
```
➜  checkstyle git:(fix-link-check) find notpresent -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;        

find: ‘notpresent’: No such file or directory
➜  checkstyle git:(fix-link-check) echo $?
1
➜  checkstyle git:(fix-link-check) find notpresent -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \; || true

find: ‘notpresent’: No such file or directory
➜  checkstyle git:(fix-link-check) echo $?                                                                  
0

```

If build is failing, what value will location of maven repo provide?

Comment (rnveach) [reply .ci/travis.sh:148]: It will tell us if maven repo changes location, which would be one cause for the failure.

Comment (nrmancuso) [reply .ci/travis.sh:148]: I would prefer to keep scripts as simple as possible, and add debugging commands as needed when a problem arises. But let's get a second opinion. @romani ?

Comment (romani) [reply .ci/travis.sh:148]: I always prefer simplicity of bash scripts. I agree with Nick.
But extra logging is also ok.

**Review by romani — APPROVED** [review]

**Review by nrmancuso — APPROVED** [review]
Ok, let's merge this.

**Comment by nrmancuso** [comment]
Failure in https://github.com/checkstyle/checkstyle/actions/runs/3453271573/jobs/5763738631 is not related to changes here.

**Pull request merged**